src/main/scala/org/allenai/deep_qa/data/BackgroundCorpusSearcher.scala

```diff
@@ -82,19 +82,39 @@ class LuceneBackgroundCorpusSearcher(params: JValue) extends BackgroundCorpusSea
   lazy val settings = Settings.builder().put("cluster.name", esClusterName).build()
   lazy val esClient = TransportClient.builder().settings(settings).build().addTransportAddress(address)
 
-  override def getBackground(query: String): Seq[String] = {
+  override def getBackground(query: Query): Seq[String] = {
+    // Make the bool query and the more-or-less canonical query string for the
+    // consolidation step (below)
+    val queryBuilder = QueryBuilders.boolQuery()
+    var queryString = ""
+    query match {
+      case sq: StringQuery => {
```

Comment:
Avoid abbreviations like this.  It's better to call this `stringQuery`, so that I immediately know what it is when I see it used below.  Code readability is way more important than short lines: https://google.github.io/styleguide/cppguide.html#General_Naming_Rules.